components/brave_vpn/browser/brave_vpn_service.cc

```diff
@@ -131,7 +136,9 @@ void BraveVpnService::BindInterface(
 void BraveVpnService::OnConnectionStateChanged(mojom::ConnectionState state) {
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
   VLOG(2) << __func__ << " " << state;
-
+#if BUILDFLAG(IS_WIN)
+  wireguard::WriteConnectionState(static_cast<int>(state));
```

Comment:
This is called for connection state change from ras. Maybe only need for wireguard?

Reply:
Hmm, I think this used for both? but it's confusing that it's in `wireguard` namespace.
Could we update namespace for methods that commonly used for both?

Reply:
sure, done